frontend/components/forms/field-info/component.tsx

```diff
@@ -13,7 +13,7 @@ export const FieldInfo = ({ infoText }: FieldInfoProps) => {
       arrow
       arrowClassName="bg-black"
       content={
-        <div className="max-w-xs sm:max-w-md p-2 font-sans text-sm font-normal text-white bg-black rounded-sm">
+        <div className="max-w-xs p-2 font-sans text-sm font-normal text-white bg-black rounded-sm sm:max-w-md">
```

Comment:
This change was automatic due to the use of the [Headwind](https://marketplace.visualstudio.com/items?itemName=heybourn.headwind) VSCode extension.